Radio.h

```diff
@@ -151,6 +152,12 @@ enum radioAtuState {
 - (void) updateWaterfallRef:(Waterfall *) waterfall;
 @end
 
+@protocol TNFEventHandler <NSObject>
+
+@optional
+- (void) tnfAdded:(Tnf *)tnf;
```

Comment:
Doug,

I'm wondering whether this should be done via notification (on the main queue) rather than via a delegate?  TNFs are of interest to view controllers displaying panafalls of which there may be more than one- conceptually up to 8 different ones.  A notification would avoid having the client to rotor out its notifications.

Also, it would be nice given a panafall object to return a list of all the TNFs currently in its bandwidth - seems like a generic enough requirement to implement as a helper function.